### PR TITLE
LG-2963, LG-3170: LiveExample links

### PR DESCRIPTION
**Pull request by spark33**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-2963](https://jira.mongodb.org/browse/LG-2963)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/banner/src/Banner.story.tsx [hunk 1]
```diff
@@ -35,12 +36,20 @@ const meta: StoryMetaType<typeof Banner> = {
...
   args: {
-    children:
-      'To avoid disrupting majority writes, new members are now added to replica sets as priority=0, votes=0 until they reach secondary state, after which Cloud Manager automatically updates the configuration to match the priority and votes value specified in the deployment.',
+    children: (
+      <>
+        To avoid disrupting majority writes, new members are now added to
+        replica sets as priority=0, votes=0 until they reach secondary state,
+        after which Cloud Manager automatically updates the configuration to
+        match the priority and votes value specified in the deployment. &nbsp;
+        <a href>Anchor tag</a>&nbsp;<Link>Link Component</Link>
```
Comment (shaneeza) [inline packages/banner/src/Banner.story.tsx:45]: I think you need a valid href, like in callout below, so that the correct cursor shows up

**Review by shaneeza — COMMENTED** [review]
I'm still seeing the issues in this [ticket](https://jira.mongodb.org/browse/LG-3170) here https://642700aa3e49e32bdbf0b0fc-etflvmitgx.chromatic.com/?path=/story/components-callout--live-example

**Review by shaneeza — APPROVED** [review]
Looks good, also looks like there are some chromatic changes for callout

**Comment by audroue** [comment]
I'm seeing one of the links act differently than the other on hover: https://jmp.sh/67CatGzC

**Pull request merged**